packages/hoppscotch-common/package.json

```diff
@@ -136,6 +136,7 @@
     "eslint": "^8.47.0",
     "eslint-plugin-prettier": "^5.0.0",
     "eslint-plugin-vue": "^9.17.0",
+    "glob": "^10.3.10",
```

Comment:
Isn't glob a dev-dependency in this case ?

Reply:
It's already present under `devDependencies`.

Reply:
oops lol. missed that due to the fold.